README.rst

```diff
@@ -156,6 +156,25 @@ with: ``mlt``, ``update``, ``terms`` etc. The specific methods of pysolr's ``Sol
 Setting a handler in ``search`` explicitly overrides the ``search_handler`` setting (if any).
 
 
+Custom Commit Policy
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    # Setup a Solr instance. The trailing slash is optional.
+    # All request to solr will result in a commit
+    solr = pysolr.Solr('http://localhost:8983/solr/core_0/', search_handler='/autocomplete', commit_by_default=True)
```

Comment:
Maybe this should be something like send_commits or something? by_default doesn't sound right to me.